engines/bastion_katello/app/assets/javascripts/bastion_katello/packages/details/package-details-repositories.controller.js

```diff
@@ -22,7 +22,7 @@ angular.module('Bastion.packages').controller('PackageDetailsRepositoriesControl
             'organization_id': CurrentOrganization
         };
 
-        repositoriesNutupane = new Nutupane(Repository, params, 'repositories');
+        repositoriesNutupane = new Nutupane(Repository, params);
```

Comment:
@waldenraines this is intentional. I was trying to fix another bug with this change. If you go to Package Search and click on Repositories you get a js error . Look at 
![Package Search](http://i.imgur.com/SkY8lcd.png)